src/Microsoft.Extensions.Options.ConfigurationExtensions/OptionsConfigurationServiceCollectionExtensions.cs

```diff
@@ -22,6 +23,31 @@ public static class OptionsConfigurationServiceCollectionExtensions
         public static IServiceCollection Configure<TOptions>(this IServiceCollection services, IConfiguration config)
             where TOptions : class
         {
+            var setups = services.Where(s => s.ServiceType == typeof(IConfigureOptions<TOptions>) && s.ImplementationInstance != null && s.ImplementationType == typeof(ConfigureFromConfigurationOptions<TOptions>));
+            var order = -10000; // REVIEW: should we expose this default constant for config?
```

Comment:
I don't think we should go back to special casing the order of configuration binding at all. Simple order of registration provides enough control and enough rules for anyone to understand :smile:. We have lived without it since beta 8 and personally I haven't seen feedback indicating that it is missed.